dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/IEC/WT4AIECelec.mo

```diff
@@ -0,0 +1,138 @@
+    Dialog(tab = "Operating point"));
+  parameter Types.ReactivePowerPu Q0Pu "Initial reactive power at plant terminal in pu (base SnRef) (receptor convention)" annotation(
+    Dialog(tab = "Operating point"));
+
+  //Circuit parameters
+  parameter Types.PerUnit ResPu "Serial resistance in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Electrical"));
+  parameter Types.PerUnit XesPu "Serial reactance in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Electrical"));
+  parameter Types.PerUnit GesPu "Shunt conductance in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Electrical"));
+  parameter Types.PerUnit BesPu "Shunt susceptance in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Electrical"));
+
+  //Control parameters
+  parameter Types.PerUnit DipMax "Maximun active current rising slew rate in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Control"));
+  parameter Types.PerUnit DiqMax "Maximun reactive current rising slew rate in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Control"));
+  parameter Types.PerUnit DiqMin "Maximum reactive current falling slew rate in pu (base UNom, SNom)" annotation(
+    Dialog(tab = "Control"));
+  parameter Types.Time tG "Current generation time constant in s" annotation(
+    Dialog(tab = "Control"));
+
+  //Interface
+  Dynawo.Connectors.ACPower terminal(V(re(start = u0Pu.re), im(start = u0Pu.im)), i(re(start = 0), im(start = i0Pu.im))) "Grid terminal" annotation(
+    Placement(visible = true, transformation(origin = {105, 0}, extent = {{-5, -5}, {5, 5}}, rotation = 0), iconTransformation(origin = {-110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  //Input variables
+  Modelica.Blocks.Interfaces.BooleanInput fOCB "Breaker position, true if closed" annotation(
+    Placement(visible = true, transformation(origin = {-1.77636e-15, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90), iconTransformation(origin = {-60, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Interfaces.RealInput ipCmdPu(start = -P0Pu * SystemBase.SnRef / (SNom * U0Pu)) "d-axis reference current at generator terminal in pu (Ubase, SNom) (generator convention)" annotation(
+    Placement(visible = true, transformation(origin = {-110, -20}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 180)));
+  Modelica.Blocks.Interfaces.RealInput ipMaxPu(start = IpMax0Pu) "Maximal d-axis reference current at generator terminal in pu (Ubase, SNom) (generator convention)" annotation(
+    Placement(visible = true, transformation(origin = {-110, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 79}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
+    Modelica.Blocks.Interfaces.RealInput iqCmdPu(start = Q0Pu * SystemBase.SnRef / (SNom * U0Pu)) "q-axis reference current at generator terminal in pu (Ubase, SNom) (generator convention)" annotation(
```

Comment:
From my discussion with Carmen, everything is coherent, even though the "-" is not everywhere, because of our interpretation of the norm, but this interpretation may change in the future.

Reply:
Ok, thanks